src/types/index.ts

```diff
@@ -167,7 +167,7 @@ export type PlaidLinkPropTypes = PlaidLinkOptions & {
 
 export interface PlaidHandler {
   open: () => void;
-  exit: (force?: boolean) => void;
+  exit: (exitConfig?: { force?: boolean }) => void;
```

Comment:
this is the fix to `exit` - you can see that `exit` expects an object rather than a boolean arg here https://github.com/plaid/react-plaid-link/pull/318/files#diff-399d91c5730fe8c956a45beb380a60404c1fcea16640f3a5f58b82e052aebfdaR77